<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysmessage</class>
 <widget class="QDialog" name="sysmessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="System_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="Action_No_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>none</string>
   </property>
  </widget>
  <widget class="QLabel" name="Danger_symbol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Action_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Action : </string>
   </property>
  </widget>
  <widget class="QLabel" name="Code_No_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="Action_symbol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Code_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Code : </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
